test/bbb_eth_listener_test/README.md

```diff
@@ -0,0 +1,23 @@
+## BBB Eth Listener
+This runs on the BBB, and connects the network table to the ethernet port.
+The ethernet port is connected to the NUC, which is running the controller/simulation
+code. This is used to transfer data to and from the controller/simulator.
+
+# Setting up 
+Of course, make sure there is a connection between the BBB and the Intel NUC.
+This is typically done via ethernet cable, although technically it is not
+required (the devices could communicate over a home wifi network as well).
+Before running, the ethernet port must appear as an active network interface.
+You can view active interfaces on the BBB with  
+```ifconfig```
+
+# Running
+Make sure to run the bbb_eth_listener first, before running the nuc_eth_listener
```

Comment:
Need to update to run bbb_eth_listener_test.
Will also update this README to note how it's being used to read mock gps data (and explain why we need to do this)